_posts/2020-08-15-Named-Graphs.md

```diff
@@ -530,10 +550,10 @@ connections that person may have in the medieval Irish genealogies
 which will help understand and read the context of the other text.
```

Comment:
I still don't think this paragraph makes sense. You should try to define more clearly when someone might want to search all the MS versions at once, without any differentiation being made in terms of where information is coming from. However, it is ultimately your blogpost so I am not going to keep insisting on this.

Reply:
I removed it.  It doesn't make much sense to you so I will have to find some other way to express this another time.

Reply:
You shouldn't remove it - please don't overreact. It is very important that you explain to people not coming to this from a tech angle why they should be getting on top of Named Graphs. Personally, I think the majority of users will want/need to know which manuscript text(s) any given piece of information is coming from. This is certainly true for anyone (like myself) coming to this from the point of view of textual or historiographical scholarship. However, you clearly believe that there will also be users who are happy to query the corpus as a whole. Perhaps they are just quickly checking to see if a statement finds support _somewhere_ in the genealogies? Or are there large-scale, statistical analyses that might also be done this way? If you set your thinking out explicitly, the user will be able to make an informed decision as to their approach.

Reply:
You shouldn't remove it - please don't overreact. It is very important that you explain to people not coming to this from a tech angle why they should be getting on top of Named Graphs. Personally, I think the majority of users will want/need to know which manuscript text(s) any given piece of information is coming from. This is certainly true for anyone (like myself) coming to this from the point of view of textual or historiographical scholarship. However, you clearly believe that there will also be users who are happy to query the corpus as a whole. Perhaps they are just quickly checking to see if a statement finds support somewhere in the genealogies? Or are there large-scale, statistical analyses that might also be done this way? If you set your thinking out explicitly, the user will be able to make an informed decision as to their approach.